charts/nginz/conf/static/zauth.acl

```diff
@@ -4,11 +4,13 @@ a (blacklist (path "/provider")
              (path "/bot/**")
              (path "/i/**"))
 
-u (whitelist (path "/access"))
-
 b (whitelist (path "/bot")
              (path "/bot/**"))
 
 p (whitelist (path "/provider")
              (path "/provider/**"))
 
+# LegalHold Access Tokens
+la (whitelist (path "/notifications")
+              (path "/assets/v3/**")
```

Comment:
Why does legalhold need access to the assets API?

Reply:
Oh I guess it makes sense to have access to it

Reply:
(to get not just text messages, but also e.g. pictures)